src/components/App.js

```diff
@@ -0,0 +1,11 @@
+import BooksList from './BooksList';
```

Comment:
* IT'S  placed in `src/containers` directory. Let's separate container and presentation components.